source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -0,0 +1,314 @@
+#include "stdafx.h"
+#include "CppUnitTest.h"
+#include "MarkDownParser.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+using namespace AdaptiveCards;
+using namespace std;
+
+namespace AdaptiveCardsSharedModelUnitTest
+{
+    TEST_CLASS(EmphasisLeftDelimterTest)
+    {
+        TEST_METHOD(LeftDelimiterTest)
+        {
+            MarkDownParser parser("*foo bar*");
+            Assert::AreEqual<string>("<p><em>foo bar</em></p>", parser.TransformToHtml());
+        }
```

Comment:
>} [](start = 8, length = 1)

Formatting: let's leave a blank line between test methods for easier readibility